chef/cookbooks/crowbar/templates/default/crowbar-upgrade-os.erb

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+#
+# This script starts the upgrade of the node server to the latest version of
+# system and Cloud product.
+
+LOGFILE=/var/log/crowbar/node-upgrade.log
+mkdir -p "`dirname "$LOGFILE"`"
+
+# Copy stdout to fd 3
+exec 3>&1
+# Create fd 4 for logfile
+exec 4>> "$LOGFILE"
+# Set fd 1 and 2 to logfile
+exec 1>&4 2>&1
+
+set -x
+
+UPGRADEDIR=/var/lib/crowbar/upgrade
+RUNFILE=/var/run/node-upgrading
```

Comment:
I'd have put that in /var/run/crowbar/ or put a crowbar prefix in the basename, but no huge deal.